messaging/src/main/java/org/axonframework/deadline/quartz/QuartzDeadlineManager.java

```diff
@@ -386,9 +370,6 @@ public QuartzDeadlineManager build() {
         protected void validate() throws AxonConfigurationException {
             assertNonNull(scheduler, "The Scheduler is a hard requirement and should be provided");
             assertNonNull(scopeAwareProvider, "The ScopeAwareProvider is a hard requirement and should be provided");
```

Comment:
Should we no validate that the serializer is not null?

Reply:
Good point, will do!